Modules/modExtendSaveAsText.ACM

```diff
@@ -85,21 +87,39 @@ Public Sub SaveAsText(objType As asObjectType, objName As String, strFilename As
   Select Case objType
     Case asObjectType.acForm, asObjectType.acReport
       Application.SaveAsText objType, objName, strFilename & ".tmp"
-      If IsUCS2(strFilename & ".tmp") Then
-        If CleanFormReportFile(strFilename & ".tmp", strFilename, Ignores) Then
-          If blnSaveAsUTF8 And IsUCS2(strFilename) Then
-            If Not ConvFileUCS2_to_UTF8(strFilename) Then 'convert file to UTF8
-              MsgBox "Unable to convert '" & objName & "' to UTF-8 encoding. Beware when committing."
-            End If
-          End If
-        Else
-          MsgBox "Something went wrong cleaning the file for " & objName & "."
+      blnUCS2 = IsUCS2(strFilename & ".tmp") ' indicates whether Application.SaveAsText produced UCS2 or ANSI output
+      If Not blnUCS2 Then
+        If Not ConvFileANSI_to_UCS2(strFilename & ".tmp") Then
+          blnFail = True
+          GoTo End_Processing
         End If
+      End If
+      ' now we have a UCS2 file stored with .tmp suffix
+      
+      If Forms("Access SVN - Options").chkReplaceCodeOnly Then
+        ' Replacing code only so there is no requirement to Clean the Form/Report Layout section
+        ' we need to use the existing file as well - ensure it is in UCS2 format
+        If Not ConvFile_to_UCS2(strFilename) Then blnFail = True: GoTo End_Processing

```

Comment:
Has this not already been done by the above, where `ConvFileANSI_to_UCS2` is called?

Reply:
No,  there are 2 files the one produced by Application.SaveAsText ('.tmp') and the one which is already in the folder without the tmp suffix.

Reply:
Ah yes, my mistake.